proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -458,8 +458,11 @@ message ActivityInfo {
         int64 last_redirect_counter = 2;
     }
 
-    // fist time activity was schedulled.
+    // First time the activity was scheduled.
     google.protobuf.Timestamp first_scheduled_time = 39;
+
+    // Last time an activity failure was recorded by the server.
+    google.protobuf.Timestamp last_attempt_complete_time = 40;
```

Comment:
Do you plan to change the replication side in a follow up PR?

Reply:
This is the first time I'm hearing about it. I checked, and there is a big discrepancy between ActivityInfo and SyncActivityTaskAttributes. I guess I will add it in the next PR, when I better understand what is that for and what are the criterias.